### PR TITLE
Update DocumenterVitepress compat in docs

### PR DESCRIPTION
**Pull request by asinghvi17**
Surprised CompatHelper didn't pick this up...

### CODE REVIEW

**Comment by avik-pal** [comment]
https://buildkite.com/julialang/lux-dot-jl/builds/2169#018f87e9-ecbf-4f8f-b31f-08487e6cea15/571-7457 is failing so I bounded it

**Comment by asinghvi17** [comment]
Ah right, forgot about that.  Will update this PR once I've pushed a fix.

File: docs/Project.toml [hunk 1]
```diff
@@ -30,4 +30,4 @@ WeightInitializers = "d49dbf32-c5c2-4618-8acc-27bb2598ef2d"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
```
Comment (avik-pal) [inline docs/Project.toml:32]:
```suggestion
```

File: docs/Project.toml [hunk 2]
```diff
@@ -30,4 +30,4 @@ WeightInitializers = "d49dbf32-c5c2-4618-8acc-27bb2598ef2d"
...
-DocumenterVitepress = "0.0.19"
+DocumenterVitepress = "0.0.19, 0.0.20"
```
Comment (avik-pal) [inline docs/Project.toml:33]:
```suggestion
```

**Review by avik-pal — CHANGES_REQUESTED** [review]

**Comment by asinghvi17** [comment]
Looks like the doc build worked at least.

**Pull request merged**